scripts/Modules/mkw_core.py

```diff
@@ -88,3 +88,31 @@ def get_speed(playerIdx=0):
     xyz = math.sqrt(x ** 2 + y ** 2 + z ** 2)
 
     return speed(x, y, z, xz, xyz)
+
+# Next 3 functions are used for exact finish display
+def getIGT(lap, player): # lap is 1-indexed
+    offsets = [0xC, (player)*0x4, 0x3C, (lap-1)*0xC + 0x5]
+    value = chase_pointer(classes.getRaceInfoHolder(), offsets, 'u16')
+    return divmod(value, 0x100)
```

Comment:
Neat trick. I think it's better to create the timer in this function instead of doing half of it here and half of it in `updateExactFinish` (especially since both timer creations are consistent).